.github/actions/setup-python-matrix/action.yml

```diff
@@ -8,6 +8,11 @@ runs:
         python-version: "pypy-3.7"
         architecture: x64
 
+    - uses: actions/setup-python@v4
```

Comment:
This whole file actually was meant to be deleted but showed back up in a rebase. Can you remove it completely?